Split Updating Guide (#2233)
theforeman/foreman-documentation: guides/common/assembly_updating-server-smart-proxy-hosts.adoc

Reviewer: But you don't use this guide to update hosts, do you?
Please reflect that in the filename + ID, i.e. `assembly_updating-server-smart-proxy.adoc`

Author: Addressed :+1:

> ifdef::context[:parent-context: {context}]
> 
> [id="Updating_Server_SmartProxy_Content_Hosts_{context}"]

Reviewer: ```suggestion
[id="Updating_Server_SmartProxy_{context}"]
```

> = Updating {ProjectServer}, {SmartProxyServerTitle}, and Content Hosts

Reviewer 2: I see content hosts listed here in the title, paragraph below and the prerequisites for `Updating Satellite Server`, but I don't see a section for them. Is the process the same as updating capsule? If so we should also add content hosts to the `Updating Capsule Server` section or if not they probably need their own section.

Author: Agree with your view. I see the content in the old guide. Either we need to have a whole new section on `Content Host` or we need to drop it completely. There is no section or mentioning of `Updating Content Host` in the old guide.

> = Updating {ProjectServer}, {SmartProxyServerTitle}, and Content Hosts

Author: Ahh, Ack! Thanks @AkshayGadhaveRH 

CC: @Griffin-Sullivan

> Use this chapter to update your existing {ProjectServer} and {SmartProxyServer} to a new patch version, for example, from {ProjectVersion}.0 to {ProjectVersion}.1.

Reviewer: ```suggestion
Use this guide to update your {ProjectServer} and {SmartProxyServer} to a new minor release version, such as from {ProjectVersion}.0 to {ProjectVersion}.1.
```

> Use this chapter to update your existing {ProjectServer}, {SmartProxyServer}, and Content Hosts to a new patch version, for example, from {ProjectVersion}.0 to {ProjectVersion}.1.

Reviewer 3: ```suggestion
Use this chapter to update your existing {ProjectServer} and {SmartProxyServer} to a new patch version, for example, from {ProjectVersion}.0 to {ProjectVersion}.1.
```

Based on what Akshay suggested a few lines above.

Author: Addressed :+1:

> Updates patch security vulnerabilities and minor issues discovered after code is released, and are often fast and non-disruptive to your operating environment.

Reviewer: ```suggestion
The minor releases patch security vulnerabilities and minor issues discovered after code is released, and are often fast and non-disruptive to your operating environment.
```

> Before updating, back up your {ProjectServer} and all {SmartProxyServers}.

Author: In my humble opinion, we should not add `Prerequisite` here, it is there in the very next sub-section. Avoiding `prereqs` after `prereqs` 

But,

I removed `To Update {ProjectServer}:` on line 13 and replaced it with `Procedure`. Got an idea of that considering your suggestion.

Thanks for this :partying_face:

> For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.

Reviewer: Perhaps we could include the backup and restore assemblies in this guide instead of refering to another guide?

Author: IMO, we should re-direct end users. Keeping backup guide in front of them may give a sense of doubt and hesitation to update.
